apps/emqx_management/test/emqx_mgmt_api_clients_SUITE.erl

```diff
@@ -30,6 +31,34 @@ init_per_suite(Config) ->
 end_per_suite(_) ->
     emqx_mgmt_api_test_util:end_suite().
 
+init_per_testcase(t_persistent_sessions = TestCase, Config) ->
+    AppSpecs = [
+        emqx_durable_storage,
```

Comment:
Nit. Should not be needed to specify that explicitly, since it's a direct dependency of `emqx`.